service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -48,13 +48,7 @@ public static String buildSubjectLineWithHearingBookingDateSuffix(final String s
             .collect(joining(","));
     }
 
-    private static String buildHearingDateText(final List<Element<HearingBooking>> hearingBookings) {
-        return " hearing " + dateFormatterService.formatLocalDateToString(
-            hearingBookingService.getMostUrgentHearingBooking(
-                hearingBookings).getStartDate().toLocalDate(), FormatStyle.MEDIUM);
-    }
-
-    private static String getFirstRespondentLastName(final CaseData caseData) {
+    public static String getFirstRespondentLastName(final CaseData caseData) {
```

Comment:
It does not make much sense to have `getFirstRespondentLastName` method in `EmailNotificationHelper` class. It was bad when method was private. Opening on API by making it public makes it even worse.

First getting last name of first responded does not lie within responsibilities of email notification helper and it adds another unnecessary reason to change helper class. Second as a developer I wound never think of looking such method in email notification helper class which makes it harder to reuse. Finally method should not take whole case data while only one small getter is used (method utilises 1/50 part of the class it gets) - method should take list of respondents and nothing more.

Reply:
I agree, I did not know this method was even here until @Gogetter mentioned it to me. Where should this method lie? In a different helper class? And yes we should pass in list of respondents rather than all of caseData.

Reply:
Could be separate helper, named right way so that it is easier to find / reuse.

Reply:
Since ticket is already in PO review stage I won't make further changes to this PR, but will make small refactor PR after merge to move this to helper method and also refactor getCasePersonalisationBuilder to use caseData. It looks like we can use caseData for most of these and then just pass in the Id from caseDetails instead of all of caseDetails